### ISSUE TITLE
Expose helper for usage with "new" QUnit API

### ISSUE THREAD

**Issue by rwjblue** [scalvert/ember-sinon-sandbox#18]
In the case where an application does not want to _always_ create a sinon sandbox around each test, it may be nice to have a helper method that works well with the new ember-qunit@3 testing API's.

I'm generally proposing:

```js
import { module, test } from 'qunit';
import { setupTest } from 'ember-qunit';
import { setupSinonSandbox } from 'ember-sinon-sandbox';

module('service:foo', function(hooks) {
  setupTest(hooks);
  setupSinonSandbox(hooks);

  test('do stuff', function() {
    // uses sandbox
  });
});
```

The implementation would be pretty trivial, since we are already setup as individual functions that handle setup and teardown:

```js
export function setupSinonSandbox(hooks) {
  hooks.beforeEach(createSandbox);
  hooks.afterEach(restoreSandbox);
};
```

Thoughts?

**Comment by scalvert** [comment]
👍 to this. I'll look at implementing

**Comment by scalvert** [comment]
I'm not sure we should do `beforeEach`/`afterEach`. Calling `createSandbox` is a bit of overhead for each test. Would it make more sense to do it on `before`/`after`?

**Comment by rwjblue** [comment]
You need a new sandbox per test, otherwise you leak weird details between tests.

**Comment by scalvert** [comment]
Also, currently the `setupSinonSandbox` function takes an `options` hash, where we pass in QUnit (this allows for clean stubbing for testing). Does the new API support any other params in the signature other than `hooks`? I presume you can do whatever you like, but the API won't be as clean/consistent/predictable.

**Comment by rwjblue** [comment]
> I presume you can do whatever you like, but the API won't be as clean/consistent/predictable

Confirm. It would be the same. But I don't see why you need `QUnit` at all.

The example code I wrote in the description should essentially "just work".  What am I missing?

**Comment by scalvert** [comment]
Oh lolz yerp.

**Comment by scalvert** [comment]
So `testStart`/`testDone` or `beforeEach`/`afterEach` 👍 
`before`/`after` 👎 

So obvious...

**Comment by scalvert** [comment]
Addressed in #19
